app/views/questions/show.html.erb

```diff
@@ -12,6 +12,11 @@
         <br>
         <div id="question-body" >
           <%= @question.body %><br>
+          <div id="delete-question">
+              <% if !current_user.nil? && current_user.id == @question.user_id %>
```

Comment:
Instead of `` !current_user.nil? `` it could just be `` current_user ``. The logic would be the same